torch/csrc/dynamo/guards.cpp

```diff
@@ -34,13 +34,17 @@ class TensorCheck {
         requires_grad_(state.grad_mode_enabled && v.requires_grad()),
         dynamic_shapes_(dynamic_shapes) {
     auto ndim = v.ndimension();
-    const auto& sizes = v.sizes();
-    const auto& strides = v.strides();
-    sizes_.reserve(ndim);
-    strides_.reserve(ndim);
-    for (auto i : c10::irange(ndim)) {
-      sizes_.emplace_back(sizes[i]);
-      strides_.emplace_back(strides[i]);
+    // FIXME: add path for nested when removing graph break
```

Comment:
should this be handled in guards.py instead? I'm not sure why it seems to be creating a TENSORMATCH guard on a nested tensor despite the graph break